QC/wdl/tasks/hapDotPy.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+workflow runhapDotPy {
+
+    call hapDotPy
+
+    output{
+        File hapDotPyVCF    = hapDotPy.vcfOut
+        File hapDotPyVCFIdx = hapDotPy.vcfIdxOut
+        File hapDotPyTar    = hapDotPy.happyTar
+    }
+}
+
+task hapDotPy{
+    input{
+        File truthVCF
+        File queryVCF
+        File assembly
+        File assemblyIndex
+        String sample
+
+        Boolean passOnly = true
+
+        Int memSizeGB = 24
+        Int threadCount = 8
+        Int diskSizeGB = 128
+        String dockerImage = "jmcdani20/hap.py@sha256:0812d37e7210011e407914deb3da2094ec8258077b21d3211e694e5ca303b489" # v0.3.12
```

Comment:
Here and in the WDLs in this PR, you allow the user to specify a Docker image. Strictly speaking that is not ideal for reproducibility. However, since you're working with bleeding edge technologies and I assume the Docker images are getting updated regularly, I think this is acceptable for the time being. But if any of these WDLs are using Docker images that are not getting updated monthly or more often, I'd recommend hardcoding the Docker image.

Reply:
Yeah, this was a very purposeful decision in this case. The PMDV task (for instance) has documentation for an older image, and I wrote the WDL for the newest one.